### DISCUSSION TITLE
作业course的疑问

### DISCUSSION THREAD

**Discussion by DM0U** [physics-data/faq#114] [Q&A] [answered]
请问环境变量`SUFFIX`随机初始化时是否可能为空或者出现重复（例如`.py`），从而生成特殊后缀名的文件？
（这似乎会影响到`clean`的写法，即是否可以使用%${S}进行匹配删除）
<br/>
另外，对于减少硬编码的要求，如果直接将所有课程按照依赖关系直接写成各个语句，是否会扣白盒分？
例如：
````
a : b c
    ...

b : c
    ...

c : d
    ...

d :
    ...
````

**Comment by heroxbd** [comment, marked as answer]
抱歉没能及时回复，我猜你已经解决了该问题。
